WebAPI/Controllers/TimetableController.cs

```diff
@@ -153,11 +153,20 @@ public async Task<IActionResult> GetCourseTimetable(string courseId, string stud
             if (_student == null)
             {
                 return ErrorResponse($"Student with id: {studentId} does not exist.");
+            }
```

Comment:
TI don't think this logic should be in the controller...
It should be inside service...

Reply:
The logic is moved now. I created new issue for removing obsolete property, because just removing it from Course.cs does not solve that. #421